lib/src/generator.dart

```diff
@@ -49,45 +54,47 @@ void _generateHeader(StringBuffer buffer, List<I18nData> allLocales) {
   bool defaultingToEn = false;
 
   // current locale variable
-  buffer.writeln('\nconst String $baseLocaleVar = \'$baseLocale\';');
+  buffer.writeln();
+  buffer.writeln('const String $baseLocaleVar = \'$baseLocale\';');
+  buffer.writeln();
   buffer.writeln('String $localeVar = $baseLocaleVar;');
 
   // map
+  buffer.writeln();
   buffer.writeln('Map<String, $baseClassName> $mapVar = {');
+
   allLocales.forEach((localeData) {
     String finalClassName = localeData.base
         ? baseClassName
         : baseClassName + localeData.locale.capitalize().replaceAll('-', '');
+
     buffer.writeln('\t\'${localeData.locale}\': $finalClassName.instance,');
   });
-  if (baseLocale == '' &&
-      allLocales.indexWhere((locale) => locale.locale == 'en') == -1) {
-    buffer.writeln(
-        '\t\'en\': $baseClassName.instance, // assume default locale is en, add a specific \'en\' locale to remove this or add config.i18n.json');
+
+  if (baseLocale == '' && allLocales.indexWhere((locale) => locale.locale == 'en') == -1) {
+    buffer.writeln('\t\'en\': $baseClassName.instance, // assume default locale is en, add a specific \'en\' locale to remove this or add config.i18n.json');
     defaultingToEn = true;
   }
+
   buffer.writeln('};');
 
   // t getter
-  buffer.writeln('\n/// Method A: Simple');
+  buffer.writeln();
+  buffer.writeln('/// Method A: Simple');
   buffer.writeln('///');
-  buffer.writeln(
-      '/// Widgets using this method will not be updated after widget creation when locale changes.');
-  buffer.writeln(
-      '/// Translation happens during initialization of the widget (method call of t)');
+  buffer.writeln('/// Widgets using this method will not be updated after widget creation when locale changes.');
+  buffer.writeln('/// Translation happens during initialization of the widget (method call of t)');
   buffer.writeln('///');
   buffer.writeln('/// Usage:');
   buffer.writeln('/// String translated = t.someKey.anotherKey;');
-  buffer.writeln('$baseClassName get t {');
-  buffer.writeln('\treturn $mapVar[$localeVar];');
-  buffer.writeln('}');
+  buffer.writeln('$baseClassName t;');
```

Comment:
Maybe initialize with `_strings[_locale]` to avoid possible NPE?

Reply:
Agree, added